rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/rest/AuthResource.java

```diff
@@ -0,0 +1,77 @@
+package io.quarkus.sample.superheroes.fight.rest;
+
+
+
+import io.quarkus.sample.superheroes.fight.client.AuthRestClient;
+import io.quarkus.sample.superheroes.fight.client.WebAuthnRegisterResponse;
+
+import io.smallrye.mutiny.Uni;
+
+import io.vertx.core.json.JsonObject;
+
+
+import jakarta.ws.rs.BeanParam;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.FormParam;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+
+import jakarta.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.logging.Logger;
+
+import static jakarta.ws.rs.core.MediaType.*;
+
+@Path("/")
+public class AuthResource {
+  private static final Logger LOG = Logger.getLogger(AuthResource.class);
+  @RestClient
+  AuthRestClient authClient;
+
+  @Path("/q/webauthn/webauthn.js")
+  @GET
+  public Uni<Response> getJavascript(){
+    LOG.info("Returning authservice javascript......");
+    return authClient.javascript();
+  }
+  @POST
+  @Path("/q/webauthn/register")
+  @Consumes(APPLICATION_JSON)
+  public Uni<Response> setUpObtainRegistrationChallenge(JsonObject jsonObject){
+    LOG.info("Returning registration challenge....");
+    LOG.info(jsonObject);
```

Comment:
This could be replaced with `Log.infof("Returning registration challenge...%s", jsonObject);`